### PR TITLE
feat: add default `DidResolver`

### PR DESCRIPTION
**Pull request by ethanwlee**
adds default `DidResolver` with ~`DidJwkResolver`~, `DidDhtResolver` and `DidWebResolver` as the default method resolvers to make `DapResolver` easier to use

### CODE REVIEW

File: lib/src/dap_resolver.dart [hunk 1]
```diff
@@ -11,8 +11,18 @@ class DapResolver {
   http.Client client;
   DidResolver didResolver;
 
-  DapResolver(this.didResolver, {http.Client? client})
-      : client = client ?? http.Client();
+  static final _defaultMethodResolvers = [
+    DidJwkResolver(),
```
Comment (mistermoe) [inline lib/src/dap_resolver.dart]: no `DidJwk` becauuuse `did:jwk` doesn't support `service` in did docs

**Review by mistermoe — APPROVED** [review]
left 1 comment. otherwise 🚀

File: lib/src/dap_resolver.dart (hunk 1 above)
Comment (ethanwlee) [reply lib/src/dap_resolver.dart]: gotcha!

File: lib/src/dap_resolver.dart [hunk 2]
```diff
@@ -11,8 +11,17 @@ class DapResolver {
...
+    DidDhtResolver(),
+    DidWebResolver(),
+  ];
+
+  DapResolver({
+    DidResolver? didResolver,
+    http.Client? client,
+  })  : didResolver = didResolver ??
+            DidResolver(methodResolvers: _defaultMethodResolvers),
+        client = client ?? http.Client();
```
Comment (jiyoonie9) [inline lib/src/dap_resolver.dart:24]: damn that's a lot of `??`

**Pull request merged**